senders/pagerduty/init.go

```diff
@@ -0,0 +1,30 @@
+package pagerduty
+
+import (
+	"time"
+
+	"github.com/moira-alert/moira"
+)
+
+// Sender implements moira sender interface for pagerduty
+type Sender struct {
+	ImageStores  map[string]moira.ImageStore
+	imageStoreID string
+	logger       moira.Logger
+	frontURI     string
+	location     *time.Location
+}
+
+// Init loads yaml config, configures the pagerduty client
+func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+	sender.frontURI = senderSettings["front_uri"]
+	sender.imageStoreID = senderSettings["image_store"]
+	if sender.imageStoreID == "" {
+		logger.Warningf("cannot read image_store from the config, will not be able to attach plot images to events")
+	} else if !(sender.ImageStores[sender.imageStoreID].IsEnabled()) {
```

Comment:
Sorry for bad comment about the right configuration of image store. I did not see that you return an error if store just not configured well. Can we remove this "IsEnabled" logic? Sorry =(

Reply:
I don't understand. Shouldn't we check if the image store is configured properly if an image store ID is specified for this sender? 
Earlier I had,
if "image_store" field doesn't exist, then return an error. (This will cause the sender to not work at all but we want it to work without the plotting capability in this case)

Now,
if "image_store" field doesn't exist, then warn the user and continue sender configuration.
if it exists, then check if it is configured, if not warn the user and continue.

The IsEnabled() is used to check if it is configured.

Reply:
Oh, I understand what the problem is. You're right.

Reply:
Let's act then in another way. It seems that putting this flag down is a leak of responsibility. Let's better make the Enabled flag private and will put it in the Init method after configuration.